rllib/env/tests/test_multi_agent_episode.py

```diff
@@ -1023,6 +1023,50 @@ def test_get_actions(self):
         act = episode.get_actions(-4, env_steps=False, fill="skip")
         check(act, {"a0": "skip", "a1": 0})
 
+        episode.add_env_step(
```

Comment:
Added this to the tests. Mostly to figure out, whether a hanging action at the edge of the episode or further back would make a difference in `get_actions(-1)`.